pre_award/assess/assessments/routes.py

```diff
@@ -1818,3 +1818,25 @@ def view_entire_application(application_id):
         application_id=application_id,
         mapped_answers=map_answers,
     )
+
+
+@assessment_bp.route(
+    "/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>/theme_id/<theme_id>/success_page",
+    methods=["GET"],
```

Comment:
Let please make the url:
```python
/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>/theme_id/<theme_id>/request_change/success
```

Basically the URL we came from with `/success` at the end.